### ISSUE TITLE
Selection not working any more on lollipop after month swipe.

### ISSUE THREAD

**Issue by fabiobond** [prolificinteractive/material-calendarview#100]
Hi,
I'm trying to integrate this beautiful library into my project but I'm encountering a very strange behaviour on Lollipop (on devices with S.O. < Lollipop all is working as expected)

So I've an activity that has a FrameLayout and the MaterialCalendarView widget.

If I select any date on the calendar, the selected day get decorated as expected.
Once I swipe/switch to another month, every time I select I day, nothing is selected.
If I just try to swipe/scroll a bit without really changing month, then the last selected day get decorated. It seem something related to redraw of the widget. 

I've tried adding a dateSelectedListener and adding a materialCalendarWidget.postInvalidate() when the event occurs but nothing change.

I've also tried to upgrade to appcompat 22.2.0 (before I was using 22.1) and deriving my activity from Activity, appCompatActivity,ActionBarActivity,ecc but nothing change.

My project uses AndroidAnnotations library, can this be incompatible with MaterialCalendarView ?

**Comment by dandc87** [comment]
Hey @fabiobond, are you using the latest version (0.7.0)? I remember seeing this a while ago, but I'm not able to reproduce this with the latest build. 

Also, could you provide me some more information on how you're using it? I.e. what options are you setting, are you using decorators, etc.

And I'm not too familiar with AndroidAnnotations, but there shouldn't be any compatibility issues with it and Material CalendarView.

**Comment by fabiobond** [comment]
Hi,

yes, I'm using the latest 0.7.0. The demo app is working as expected on same phone.

I've just tried to put it in my xml layout without setting anything and without performing any setting by code.

AndroidAnnotations generate code at compile time and create instrumented classes. I've also tried not using androidannotations with that activity (but still used in project) but same behaviour occur.
I'm also using EventBus library in that project.

**Comment by quentin41500** [comment]
@fabiobond long time since this issue has been open, did it get resolved with the new versions? If so, I will close this issue.

**Comment by fabiobond** [comment]
Yes, you can close the issue. I'm not experiencing it any more. Thank you



2016-05-06 20:48 GMT+02:00 Quentin Colle <notifications@github.com>:

**Comment by nmilcoff** [comment]
Hi, thank you for this great library! I'm experiencing this issue in v1.4.2 in Lollipop devices / simulator. I'm using the default configurations, I would really appreciate any hint

**Comment by jitarun** [comment]
Hi , 
I'm also facing this issue in v1.4.3 in Lollipop device (android version 5.1) using the default configuration.

**Comment by quentin41500** [comment]
Reopening as it seems to still be an issue.
